starboard/player.h

```diff
@@ -144,7 +144,7 @@ typedef enum SbPlayerSampleSideDataType {
   // The first 8 bytes of the data contains the value of BlockAddID in big
   // endian format, followed by the content of BlockAdditional.
   kMatroskaBlockAdditional,
-#if SB_API_VERSION >= 16
```

Comment:
Do you want to just remove that code? We haven't shipped SB16 yet.

Reply:
Sure, that'll be cleaner